site/assets/scss/_toc.scss

```diff
@@ -29,10 +30,18 @@
     }
 
     a {
+      padding: .1rem .2rem;
       color: inherit;
+      text-decoration: none;
+      @include border-radius(.2rem);
 
-      &:not(:hover) {
-        text-decoration: none;
+      &:hover,
+      &.active {
+        color: $bd-purple;
```

Comment:
Need to be updated with new CSS var introduced by dark mode

Reply:
Yep. Agree with you. Updated the PR to use custom properties to make it work with the new color mode (see https://github.com/twbs/bootstrap/pull/33428/commits/523b75be164262d272b124ce46d900a68ee6e76b)